package.json

```
@@ -299,6 +299,7 @@
     "solid-styled-components": "0.28.5",
     "solid-transition-group": "0.3.0",
     "ts-morph": "25.0.1",
+    "virtua": "^0.40.3",
```

```suggestion
    "virtua": "0.40.3",
```

Please use **-E** option like `pnpm i -E virtua`